packages/utils/telemetry-utils/package.json

```diff
@@ -137,10 +137,6 @@
 		"typescript": "~5.1.6"
 	},
 	"typeValidation": {
-		"broken": {
```

Comment:
This one is expected, from https://github.com/microsoft/FluidFramework/pull/20734.